src/components/Header.tsx

```diff
@@ -40,13 +43,33 @@ const Header = (): React.ReactElement => {
           }}
         />
         <Navigation.Item
-          active={active === 'accessTokens'}
+          active={active === 'apiAccessTokens'}
+          as="button"
+          label="Hae API access token"
+          type="button"
+          onClick={(): void => {
+            setActive('apiAccessTokens');
+            history.push('/apiAccessTokens');
+          }}
+        />
```

Comment:
There's a lot of repetition on these Navigation.Item elements (4 instances). Only the pageName and label change. Could perhaps use some kind of a loop for constructing these...

Reply:
> The UI tabs don't remember their state, when switching between them. This can be seen with those tabs that have user modifiable state, i.e. `apiAccessTokens` and `userTokens`. Modify some data, switch to another tab, and come back and the tab resets to default state.
> 
> However I would suggest not to fix this problem within this PR, because this is already quite a big bunch.

At the moment the whole UI is just technical demo without any actual plans, so no extra time has been devoted to UX and end user happiness.

Reply:
> There's a lot of repetition on these Navigation.Item elements (4 instances). Only the pageName and label change. Could perhaps use some kind of a loop for constructing these...

Yes, there should be one location where page names and paths are listed, because App.tsx also uses routes listed in Header.tsx. I havent done it for the same reason as above: no plans have been made and it is just a tech demo for now.